app/views/curation_concerns/file_sets/media_display/_audio.html.erb

```diff
@@ -0,0 +1,40 @@
+<% if Rails.configuration.wowza_enabled
+  solr_document = file_set
+  solr_document = file_set.solr_document if (file_set.respond_to? :solr_document)
+
+  parent_id = parent.id if defined?(parent) && !parent.nil?
+  fedora_file = solr_document['digest_ssim'].first.split(':')[-1] if !solr_document['digest_ssim'].nil?
+
+  file_ext = 'mp3'
+  file_ext = solr_document['label_tesim'].first.split('.')[-1] if solr_document['label_tesim'].first
+
+  wowzaURL = grabWowzaURL(parent_id, file_set.id, fedora_file, file_ext, '127.0.0.1')
+puts "#{parent_id} #{file_set.id} #{fedora_file} #{file_ext} 127.0.0.1: #{wowzaURL}"
+%>
+  <% if wowzaURL != nil %>
+	<div id="dams-audio">Loading the player...</div>
+
+	<script type="text/javascript">
+		jwplayer("dams-audio").setup({
```

Comment:
can we clean up the spacing here? Also the configuration may need to be updated based on the issues OPS noted in https://github.com/ucsdlib/damspas/issues/271. Depending on when @VivianChu works on that ticket, that could either get folded into this or as a separate ticket. But I want to make sure we don't lose those changes.